src/components/Vaults.ts

```
@@ -334,10 +336,12 @@ export default class Vaults {
 
   public getSingleCollateralVaults = async (
     account: string,
-    network: SingleCollateralVaultNetwork,
+    network: Network,
     signer: ethers.Signer
-  ): Promise<SingleCollateralVault[]> => {
+  ): Promise<SingleCollateralVault[] | undefined> => {
     this.initialisationCheck();
+    if (network === "ethereum") return;
+
```

Why the `if`?
Also, this should throw, not return undefined

---

refactored